src/stores/__tests__/orderListStore.spec.ts

```diff
@@ -34,24 +34,38 @@ describe('orderList store', () => {
       done();
     });
 
-    it('should be added into order list', () => {
+    it('should be added into order list and return true', () => {
       expect(orderListStore.allOrders.length).toBe(1);
-      orderListStore.addOrder({
+      const isAdded = orderListStore.addOrder({
         AssetPairId: 'BTCUSD',
         CreateDateTime: '2018-01-17T07:17:40.84Z',
-        Id: '1f4f1673-d7e8-497a-be00-e63cfbdcd0c7',
+        Id: '1f4f1673-d7e8-497a-be00-e63cfbdcd0c6',
         OrderAction: 'Sell',
         Price: 1,
         Status: 'InOrderBook',
         Volume: 0.0001
       });
       expect(orderListStore.allOrders.length).toBe(2);
+      expect(isAdded).toBeTruthy();
```

Comment:
i do not agree
why should it return `boolean`?

you were asking to return newly created `Order` from the API side, but do not do the same thing on the client side

i believe it should return new order list (`Order[]`) and then the caller will decide

Reply:
If returning boolean is not suited for this case I think it is better to return single order but not the whole list. The caller should not work with list